### DISCUSSION TITLE
Issue with Code Chunk when using Tachyons CSS Framework

### DISCUSSION THREAD

**Discussion by jmgirard** [quarto-dev/quarto-cli#1419] [Q&A] [answered]
> RStudio 2022.07.0+548 "Spotted Wakerobin" Release (34ea3031089fa4e38738a9256d6fa6d70629c822, 2022-07-06) for Windows
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.8 Chrome/69.0.3497.128 Safari/537.36

I am using the [Tachyons CSS framework](https://tachyons.io/) but this seems to be causing some problems with decimals in my R code chunks. I'm obviously not using the Tachyons classes in the MWE, but I am using them a lot in my document elsewhere. Any ideas on how I can fix this problem?

    ---
    format: 
      html:
        css: https://unpkg.com/tachyons@4.12.0/css/tachyons.min.css
    ---
    
    ```{r}
    sqrt(3.1)
    ```

When rendered, instead of displaying `sqrt(3.1)` as expected/desired, it displays `3.1sqrt()`.

![image](https://user-images.githubusercontent.com/6829033/179318671-bca1189e-2100-4644-a43e-6710b1868de6.png)

**Comment by arvindvenkatadri** [comment]
tachyons.css.min has three CSS classes pertaining to code. These change the font and the formatting of code chunks in a Quarto Markdown document. I just commented out the word "code" in that CSS file and order has been restored!

**Reply by gadenbuie** [reply, marked as answer]
Interestingly, I recently used Tachyons in my slides and went through all of pandoc's code classes and came up with some Sass that you can include to suppress a few of the Tachyons classes that conflict with pandoc's styles.

Instead of just posting the snippet here, I made a Quarto extension that provides Tachyons for Quarto HTML documents: https://pkg.garrickadenbuie.com/quarto-tachyons/
